src/compactor_executor.rs

```diff
@@ -149,16 +150,22 @@ impl TokioCompactionExecutorInner {
                 _ => raw_kv,
             };
 
-            // Add to SST
-            let value = kv.value.into_option();
-            current_writer
-                .add(
-                    kv.key.as_ref(),
-                    value.as_ref().map(|b| b.as_ref()),
-                    kv.attributes,
-                )
-                .await?;
-            current_size += kv.key.len() + value.map_or(0, |b| b.len());
+            // When we compact to the last sorted run, all tombstone space is physically deleted
+            if compaction.is_dest_last_run && kv.value.is_tombstone() {
```

Comment:
The reason I chose the current processing is that we will introduce `CompactionFilter`, but it's not clear to me how @agavra will integrate `CompactionFilter` with slateDB. 

It seems to me that the name `include_filters` might imply that we are including `CompactionFilter` here, which at least at this point isn't appropriate, and I think it's a matter of two orthogonal issues (is_last_run and compaction_filter).

Reply:
This makes sense to me 👍 we can think about how to turn it into a compaction filter when we introduce that API